Server/src/GameEngine/Application/Application.hpp

```diff
@@ -55,9 +56,9 @@ class Application {
          */
         void _packetHandler(Network::UDPPacket packet, const asio::ip::udp::endpoint& endpoint);
 
-        std::shared_ptr<Network::Server>    _server;    // Network class for server.
-
-        ECS::Registry                       _registry;  // registry class for ECS management.
+        std::shared_ptr<std::vector<ECS::Registry>>             _registries;        // vector of registries class for ECS management.
+        std::shared_ptr<SceneManager::ServerSceneManager>       _sceneManager;      // load and handle scene in the ECS.
```

Comment:
Why is this a shared pointer ?

Reply:
It's because it don't has a default constructor.

Reply:
Since it isn't an interface yet, you can remove the shared pointer and it'll change legit nothing. Even if the class à no default constructor, since it has a constructor (default or not) then it's not an interface, so you can remove the shared pointer :thumbsup:

Reply:
That's mb